x11-themes/numix-gtk-theme/numix-gtk-theme-9999.ebuild

```diff
@@ -1,33 +1,38 @@
 # Copyright 1999-2015 Gentoo Foundation
```

Comment:
Please update the copyright since you are updating the ebuild.

Reply:
Done here and in numix-icon-theme-circle.